### DISCUSSION TITLE
Social Share Bar Component

### DISCUSSION THREAD

**Discussion by efonsecab** [microsoft/fluentui-blazor#2870] [Ideas]
I think it would come handy to have a Social Share bar component such as the one I created for my FairPlay platform
![image](https://github.com/user-attachments/assets/68dd0533-f169-41e3-a5f1-fbcf3d648efd)

If it is useful I could implement it myself, what I'm not sure about is the icons, for the FairPlay platform I had to use some bootstrap icons because I did not find social platform icons in the fluent ui library

@vnbaaij what do you think about adding this kind of component?

**Comment by vnbaaij** [comment]
I think this would be a good component for the community package that was spoken of before. 
I would not want to have this in the core package. Mainly because of the icons which could give issues with regards to copyrights. That is also the reason why those are not in the Fluent Icon collection.

With regards to the community package, that is really something that the community needs to pick up. If we organize this, we would still be maintainers.
